src/components/NonAuthenticatedRoutes/index.jsx

```diff
@@ -10,28 +10,27 @@ import AfterRegister from 'pages/Register/AfterRegister';
 import RegisterConfirm from 'pages/Register/RegisterConfirm';
 import About from 'pages/About';
 
-const NonAuthenticatedRoutes = ({ component: Component, ...rest }) =>
-    console.log(`nonauthenticated: ${rest.authenticated}`) || (
-        <Route
-            {...rest}
-            render={() =>
-                rest.authenticated ? (
-                    <Redirect to="/main" />
-                ) : (
-                    <Fragment>
-                        <Switch>
-                            <Route exact path="/login" component={Login} />
-                            <Route path="/about" component={About} />
-                            <Route path="/register" component={Register} />
-                            <Route path="/afterRegister" component={AfterRegister} />
-                            <Route path="/registrationConfirm/:token" component={RegisterConfirm} />
-                            <Redirect from="/*" to="/login" />
-                        </Switch>
-                    </Fragment>
-                )
-            }
-        />
-    );
+const NonAuthenticatedRoutes = ({ component: Component, ...rest }) => (
+    <Route
+        {...rest}
+        render={() =>
+            rest.authenticated ? (
+                <Redirect to="/main" />
+            ) : (
+                <Fragment>
+                    <Switch>
+                        <Route exact path="/login" component={Login} />
+                        <Route path="/about" component={About} />
+                        <Route path="/register" component={Register} />
+                        <Route path="/afterRegister" component={AfterRegister} />
```

Comment:
I don't think we should name routes in `camelCase`. According to SO, it should be `hyphenated`, so `after-register`. Also that helps Google bot a bit ;)

Reply:
Sure, didn't know about that ;) i will change then in every place to hyphenated ones.